app/src/main/java/com/rarilabs/rarime/modules/main/MainScreen.kt

```diff
@@ -93,12 +99,12 @@ fun AppLoadingScreen() {
     val scale = remember { mutableFloatStateOf(1f) }
 
     LaunchedEffect(Unit) {
-//        while (true) {
-//            scale.floatValue = 1.1f
-//            delay(500)
-//            scale.floatValue = 1f
-//            delay(500)
-//        }
+        while (true) {
+            scale.floatValue = 1.1f
+            delay(500)
+            scale.floatValue = 1f
+            delay(500)
+        }
```

Comment:
what's the purpose of this?

Reply:
if it's intentional, let's leave an explanation comment

Reply:
This is for unboarding animation.